rust-runtime/aws-smithy-types/src/body.rs

```diff
@@ -59,7 +59,11 @@ impl Debug for SdkBody {
 enum BoxBody {
     // This is enabled by the **dependency**, not the feature. This allows us to construct it
     // whenever we have the dependency and keep the APIs private
-    #[cfg(feature = "http-body-0-4")]
+    #[cfg(any(
+        feature = "http-body-0-4-x",
+        feature = "http-body-1-x",
+        feature = "rt-tokio"
```

Comment:
Why does `rt-tokio` require this?

Reply:
rt-tokio is what enables `PathBody`, path body uses it